Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/services/SuitService.java

```diff
@@ -58,4 +62,75 @@ public SuitDTO addSuit(SuitDTO suitDTO) {
         mapper.map(suit, suitDTO);
         return suitDTO;
     }
+
+    public String generateStream(Long suitId, List<Long> CaseIds) throws IOException {
+        Suit suit = suitDAO.findOne(suitId);
+
+//        Suit suit = new Suit("Suit name", "description");
+//
+//        List<Case> caseList = new ArrayList<>();
+//
+//        Case caze1 = new Case();
+//        caze1.setPriority(1);
+//        caze1.setDescription("Case1 description");
+//        caze1.setId((long) 1);
+//
+//        Step step1 = new Step();
+//        step1.setDescription("step 1");
+//        step1.setType(StepType.GIVEN);
+//
+//        Step step2 = new Step();
+//        step2.setDescription("step 2");
+//        step2.setType(StepType.WHEN);
+//
+//        List<Step> stepList1 = new ArrayList<>();
+//        stepList1.add(step1);
+//        stepList1.add(step2);
+//        caze1.setSteps(stepList1);
+//
+//        caseList.add(caze1);
+//
+//        Case caze2 = new Case();
+//        caze1.setPriority(1);
+//        caze1.setDescription("Case2 description");
+//        caze2.setId((long) 2);
+//
+//        Step step3 = new Step();
+//        step3.setDescription("step 3");
+//        step3.setType(StepType.GIVEN);
+//
+//        Step step4 = new Step();
+//        step4.setDescription("step 4");
+//        step4.setType(StepType.WHEN);
+//
+//        List<Step> stepList2 = new ArrayList<>();
+//        stepList2.add(step3);
+//        stepList2.add(step4);
+//        caze1.setSteps(stepList2);
+//
+//        caseList.add(caze2);
+//
+//        suit.setCases(caseList);
+
+        StringBuilder output = new StringBuilder("Feature: ");
```

Comment:
Let's use a template here. [Freemarker](http://freemarker.org/) is a pretty handy template processing lib.